programs/cleos/main.cpp

```diff
@@ -2604,7 +2608,6 @@ int main( int argc, char** argv ) {
    auto setActionPermission = set_action_permission_subcommand(setAction);
 
    // Transfer subcommand
-   string con = "eosio.token";
```

Comment:
just change `con` variable value to constant instead of string. it's default value for `-c` option, which can be changed